File: app/src/main/java/com/zuk0/gaijinsmash/riderz/ui/activity/main/fragment/trip/TripFragmentProvider.java
package com.zuk0.gaijinsmash.riderz.ui.activity.main.fragment.trip;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class TripFragmentProvider {
    @ContributesAndroidInjector(modules = TripFragmentModule.class)
    abstract TripFragment provideTripFragment();
}

File: app/src/main/java/com/zuk0/gaijinsmash/riderz/ui/activity/main/fragment/trip/TripFragmentProvider.kt
package com.zuk0.gaijinsmash.riderz.ui.activity.main.fragment.trip

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class TripFragmentProvider {
    @ContributesAndroidInjector(modules = [TripFragmentModule::class])
    abstract fun provideTripFragment(): TripFragment?
}